css/style.css

```diff
@@ -109,3 +111,181 @@ html {
     margin-left: 1rem;
   }
 }
+.hug {
+  text-align: center;
+}
+.hug__img {
+  height: 40px;
+  width: 40px;
+  left: 144px;
+  top: 4px;
+  border-radius: 32px;
+  padding: 8px;
+}
+
+.recent__work {
+  text-align: center;
+}
+.recent__work--text {
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0em;
+  Fill: Solid;
+  color: #172B4D;
+  box-sizing: border-box;
+  height: 97px;
+  width: 327px;
+}
+
+.works {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  justify-content: center;
+  grid-gap: 16px;
+}
+
+.work {
+  border-radius: 8px;
+  margin: 20px;
+  border: 1px solid #D0D9D4;
+  height: 100%;
+}
```

Comment:
- Hey, well done with your design, however, the items in the card are moving out of the card which is not what you want to achieve. This is because your container `.work` is having a default display of block which also is a requirement for the items to be aligned using flex. To fix this, add a display of flex and flex-direction of a column to adjust it.